app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context.rb

```diff
@@ -0,0 +1,190 @@
+require 'hawkularclient'
+
+class ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture
+  class CaptureContext
```

Comment:
I like this way to encapsulate the capturing.  @blomquisg We should think about copying this pattern.